server/src/main/java/dev/slimevr/bridge/ProtobufBridge.java

```diff
@@ -81,11 +82,6 @@ public void dataRead() {
 			hadNewData = true;
 		}
 		if (hadNewData && hmdTracker != null) {
-			// If HMD was DISCONNECTED, need to rebuild the skeleton, as it
-			// ignores the HMD if it's DISCONNECTED in order to support using
-			// other trackers on head.
-			if (hmd.getStatus() == dev.slimevr.tracking.trackers.TrackerStatus.DISCONNECTED)
-				Main.getVrServer().updateSkeletonModel();
```

Comment:
Why was this removed? It doesn't seem too wrong but I'm curious of an explanation

Reply:
It got moved to the tracker class directly, so it’s done for every tracker, not just HMD.

Reply:
Ah, was thinking that, cool